src/pipeline/tasks/03_train_model.py

```
@@ -0,0 +1,57 @@
+    X = df_features[features]
+    y = df_features['result']
+
+    best_params = {'gamma': 0.2, 'learning_rate': 0.01, 'max_depth': 3, 'n_estimators': 200}
+
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42, stratify=y)
+    eval_model = xgb.XGBClassifier(objective='multi:softprob', num_class=3, seed=42, **best_params)
+    eval_model.fit(X_train, y_train)
+    y_pred_proba = eval_model.predict_proba(X_test)
+    loss = log_loss(y_test, y_pred_proba)
+    accuracy = accuracy_score(y_test, eval_model.predict(X_test))
+    print(f"\n--- Final Model Performance Check ---")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)